lib/msal-common/docs/Response.md

```diff
@@ -0,0 +1,9 @@
+MSAL will return an `AuthenticationResult.ts` object as a response to all public request APIs:
```

Comment:
"to all acquire token APIs". (doesn't apply to get accounts, remove accounts etc)

Reply:
Changed it.